linetools/isgm/utils.py

```diff
@@ -528,3 +528,35 @@ def group_coincident_compoments(comp_list, output_type='list'):
         return output_dict
 
 
+def complist_to_joevp(comp_list, specfile, outfile):
+    """ From a given component list, it produces an
+    input file for JOEVP (Voigt profile fitter).
+
```

Comment:
Note that my code is called (for now) 'joebvp'.  Not to be confused with 'joe_vp', which was written by Joe Meiring.  I really don't care about this, but just thought I'd mention it.  Easily refactored, of course, if you so choose.